dev/wlp-jakartaee-transform/rules/jakarta-xml-dd.properties

```diff
@@ -43,3 +43,5 @@ javax.faces.WEBAPP_CONTRACTS_DIRECTORY=jakarta.faces.WEBAPP_CONTRACTS_DIRECTORY
 javax.faces.CLIENT_WINDOW_MODE=jakarta.faces.CLIENT_WINDOW_MODE
 javax.faces.SERIALIZE_SERVER_STATE=jakarta.faces.SERIALIZE_SERVER_STATE
 
+javax.resource.spi.security.PasswordCredential=jakarta.resource.spi.security.PasswordCredential
```

Comment:
This particular one was moved earlier in the file to line up with other ones.  You may want to remove the duplicate.

Reply:
Moved this up.